### PR TITLE
changed the layout of the header

### LINKED ISSUE #113: ✨ Make the header a bit nicer
## Description
I'd like for the header to be a bit smaller and to require less screen estate. It should also include the filtering as part of the header (which should also be redesigned)

### PR DESCRIPTION
**Pull request by kev1nboer**
Revamped the complete header. It's now a bit smaller and more organised. 

Closes #113

### CODE REVIEW

**Comment by kev1nboer** [comment]
@damcclean It looks like the logo for this year's Hacktoberfest has been updated. I think this needs to be updated on here as well? I've added a feature request but temporary closed it for your approval.

**Comment by damcclean** [comment]
Ah ok, I thought we had the updated logo. Feel free to add the new logo as part of this pull request if you want 😄

File: index.html [hunk 1]
```diff
@@ -7,13 +7,25 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <meta name="description" content="Hacktoberfest Finder helps you find issues which you could help fix." />
-    <meta name="keywords"content="Hacktoberfest, Programming, GitHub, DigitalOcean, Twilio"/>
+    <meta
+      name="description"
+      content="Hacktoberfest Finder helps you find issues which you could help fix."
+    />
+    <meta
+      name="keywords"
```
Comment (damcclean) [inline index.html]: Would you mind making these meta tags one-liners? It's just a personal preference of mine.

File: index.html [hunk 2]
```diff
@@ -7,13 +7,25 @@
...
+      content="Hacktoberfest, Programming, GitHub, DigitalOcean, Twilio"
```
Comment (damcclean) [inline index.html]: These sponsors need updated for this year. Would you mind updating those here too?

File: index.html [hunk 3]
```diff
@@ -44,33 +56,30 @@
...
+        <div class="flex site-name">
+          <img
+            src="/assets/images/header.png"
+            alt="Hacktoberfest"
+            width="304"
+          />
+          <h1
+            class="text-primary text-5xl headline font-thin tracking-wide font"
           >
+            Issue Finder
```
Comment (damcclean) [inline index.html]: Would you mind making this bolder so it stands out?

**Comment by kev1nboer** [comment]
@damcclean No problem! I've updated the code.

**Comment by damcclean** [comment]
Hey, thanks for your contribution!

**Comment by damcclean** [comment]
Oh. One quick thing before I merge, would you mind adding yourself to our contributors file, in the 2019 section 😄

**Comment by damcclean** [comment]
Thanks for that!

**Pull request merged**